app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/yourinfo/YourInfoTab.kt

```diff
@@ -224,18 +224,21 @@ internal fun YourInfoTab(
         }
       }
     }
-    Spacer(Modifier.height(8.dp))
 
+    Spacer(Modifier.height(8.dp))
     CoverageRows(coverageItems, Modifier.padding(horizontal = 16.dp))
-    Spacer(Modifier.height(16.dp))
-    CoInsuredSection(
-      coInsuredList = coInsured,
-      contractHolderDisplayName = contractHolderDisplayName,
-      contractHolderSSN = contractHolderSSN,
-      allowEditCoInsured = allowEditCoInsured,
-      onMissingInfoClick = onMissingInfoClick,
-      modifier = Modifier.padding(horizontal = 16.dp),
-    )
+
+    if (allowEditCoInsured) {
```

Comment:
Yup also makes sense to hide all of it too. I did the minimum change here before just to keep people from going in to the coinsured flow when fflag was off.